multibranch/pom.xml

```diff
@@ -88,7 +88,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>cloudbees-folder</artifactId>
-            <version>5.2.1</version>
+            <version>5.8-SNAPSHOT</version>
```

Comment:
I suppose this is just so that `MultiBranchProjectDescriptor` and thus our own `DescriptorImpl` will implement `getCategoryId()`? If so, would it not be simpler to just implement `getCategoryId()` here directly, at least until the ID is defined by an API in `cloudbees-folder`?